### DISCUSSION TITLE
fooocus is crashing while generating image

### DISCUSSION THREAD

**Discussion by 313ramsingh** [lllyasviel/Fooocus#2771] [Q&A] [answered]
fooocus is crashing while generating image, only if laptop/Pc charger is in plugged in condition, if not in charging mode, then it's generating properly is this is my system issue or issue is in fooocus model? using 3050 6gb VRAM and 12 GB RAM
![Screenshot 2024-04-17 230412](https://github.com/lllyasviel/Fooocus/assets/44858054/0d588029-fb5e-477f-a0b6-a6e7652af34e)
![Screenshot 2024-04-17 230514](https://github.com/lllyasviel/Fooocus/assets/44858054/403a6f4a-985c-47d9-aeba-b2e996380870)

**Comment by mashb1t** [comment, marked as answer]
Please check out https://github.com/lllyasviel/Fooocus/issues/1327#issuecomment-1856746547 and the other already reported issues with a similar topic at https://github.com/search?q=repo%3Alllyasviel%2FFooocus+CUDA_LAUNCH_BLOCKING&type=issues.
If this doesn't help you can also open a new issue and provide further information there.
EDIT I assume this has something to do with power consumption and/or the battery saving features of your laptop. Please check the settings regarding GPU usage in battery/plugged in configuration.
